packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "0.57.0",
+  "version": "0.57.0-fb-issuesdesigner-story3-6800.5",
```

Comment:
Yes, sounds good! Will do at last once all CR changes and bug fixes are in.
Since the latest v. is 0.58.0, should this v. be 0.58.1? or should it be 0.59.0? Thanks!

Reply:
Since this PR has new functionality, we will want it to be a minor version number bump (i.e. 0.X.0). If it was just a bug fix type of PR then we bump the patch version (i.e. 0.0.X). Since we don't yet know what this PR will end up on, I usually just put TBD in the release notes for now and then give it the final version when we start the merge process.